Plugins/Experimental/EndCombatRoundAfterSpellcast.cpp

```diff
@@ -0,0 +1,24 @@
+#include "nwnx.hpp"
+
+#include "API/CNWSCombatRound.hpp"
+
+namespace Experimental {
+
+    using namespace NWNXLib;
+    using namespace NWNXLib::API;
+
+    void EndCombatRoundAfterSpellcast() __attribute__((constructor));
+    void EndCombatRoundAfterSpellcast()
+    {
+        if (!Config::Get<bool>("END_COMBATROUND_AFTER_SPELLCAST", false))
+            return;
+
```

Comment:
A `LOG_INFO("EXPERIMENTAL: ...");` here like the others have would be nice!

Reply:
Done!